src/sentry/utils/audit.py

```diff
@@ -31,6 +32,12 @@ def create_audit_entry(
 ) -> AuditLogEntry:
     user = kwargs.pop("actor", request.user if request.user.is_authenticated else None)
     api_key = get_api_key_for_audit_log(request)
+    org_auth_token = get_org_auth_token_for_audit_log(request)
+
+    # We do not keep any user/key ID for this case for now, but only pass the hashed token itself as a label
+    # Without this, the AuditLogEntry fails on save because it cannot find an actor_label
+    if org_auth_token:
+        kwargs["actor_label"] = org_auth_token.token_hashed
```

Comment:
Is it ok to display the hashed token in the UI? The `actor_label` attribute is returned in API responses, and sometimes formatted into audit log message text.

Reply:
hmm, good point 🤔 maybe we just use the DB id here - I guess that is safer?

Reply:
Do customers have access to the `id` attribute? or will they associate that with a token they made? If tokens have a user-friendly name or alias we could show that.

Reply:
actually, good point, we can just use the token name! Makes much more sense 👍 I just blindly based this on the original API tokens, which do not have an identifier like this.